src/components/charts/SummaryPieChart/SummaryPieChart.css

```diff
@@ -49,4 +49,8 @@
 
 #recharts_measurement_span {
   display: none;
-}
\ No newline at end of file
+}
+
+.summary-pie-chart__toggle {
+  float: right;
```

Comment:
shouldn't it be `left`?

Reply:
Labels in pie chart are right aligned so I just make this "show more" also right aligned. :p 
But I just found `float` doesn't work so I changed to `text-aligned`